cmd/trade.go

```diff
@@ -192,30 +192,32 @@ func init() {
 		)
 		// --- end initialization of objects ---
 
-		log.Printf("validating trustlines...\n")
-		validateTrustlines(client, &botConfig)
-		log.Printf("trustlines valid\n")
+		l.Info("validating trustlines...")
+		validateTrustlines(l, client, &botConfig)
+		l.Info("trustlines valid")
 
 		// --- start initialization of services ---
 		if botConfig.MonitoringPort != 0 {
 			go func() {
-				e := startMonitoringServer(botConfig)
+				e := startMonitoringServer(l, botConfig)
 				if e != nil {
-					log.Println()
-					log.Printf("unable to start the monitoring server or problem encountered while running server: %s\n", e)
+					l.Info("")
+					l.Info("unable to start the monitoring server or problem encountered while running server:")
+					l.Errorf("%s", e)
```

Comment:
this should typically be in the same line as where we wrap it (i.e. the line above), and should have a newline after (like we do on the left hand side of this diff). ok for now.